webapp/channels/src/components/admin_console/license_settings/enterprise_edition/enterprise_edition_right_panel.tsx

```diff
@@ -29,7 +28,19 @@ const EnterpriseEditionRightPanel = ({
         'And more...',
     ];
 
-    const isEnterpriseOrE20 = isEnterpriseLicense(license);
+    const enterpriseToAdvancedAdvantages = [
```

Comment:
@marianunez I guess these means these won't be translated? Or do we typically not worry about that in the system console?

Reply:
Unfortunately, these weren't originally translated so didn't include that as part of this PR

Reply:
Fixed now. Changed it to be translatabe